src/components/WebexInterstitialMeeting/WebexInterstitialMeeting.scss

```diff
@@ -17,4 +17,9 @@
   .interstitial-media {
     flex: 1;
   }
+
+  .interstitial-media-container {
```

Comment:
put this before its child (.intertitial-media). And I guess it needs display flex, as the child has flex 1.